js/repeater-list.js

```diff
@@ -379,7 +379,7 @@
 			this.viewOptions.list_rowRendered({
 				container: $tbody,
 				item: $row,
-				rowData: data
+				rowData: data.items[index]
```

Comment:
this can be simplified to just `subset[index]`  I was already passing in the correct "items" array as the subset, I just mistakingly put the wrong argument there >.<